### ISSUE TITLE
MNFAIR stuck in transmit

### ISSUE THREAD

**Issue by mgenelins** [mgenelins/w0yc-repeaters#44]

**Comment by mgenelins** [comment]
Dan K0FCC turned off the radio and TNC on Monday around 9am CT. The stuck on the air signal remains.

**Comment by mgenelins** [comment]
Matthew Kaskavitch <mattkaskavitch@gmail.com>
Friday, October 4 2024 at 9:46 AM CDT

Matt,

I hunted down that signal on 145.675 the other day and it's coming from near University Ave SE and 27th SE in Prospect Park. Given it's being heard all over the metro, whatever it is must be pretty high up with some decent power. I tracked it down to a construction site at the corner I provided. The building is quite tall and there is a large crane, but it is hard to confirm - though all bearings I took pointed to that building as I walked around it. However, I'm fairly certain the transmitter is in that general area of Prospect Park as I could hear it with no antenna and just placing my pinky on the SMA connector.  

There is supposedly a 145.67 packet site in Prospect Park listed as MNPLS (WD0HWT-10) just a couple blocks north of there - not sure if it's on the air or a valid site anymore though? Certainly would be my pick to check on that site if it's still on the air given the readings I had in the area.

I did inform a couple other experienced fox hunters - and the entire MHR Discord - to also check it out.

Hope this helps!
